airbyte-proxy/Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN apt-get update -y && apt-get install -y apache2-utils && rm -rf /var/lib/apt
 # This variable can be used to update the destintion containers that Nginx proxies to.
 ENV PROXY_PASS_WEB "http://airbyte-webapp:80"
 ENV PROXY_PASS_API "http://airbyte-server:8001"
+ENV CONNECTOR_BUILDER_SERVER_API "http://airbyte-connector-builder-server:80"
```

Comment:
requests will come on port 8003, but will be forwarded to the port 80 on the connector-builder-server